exchanges/coinbasepro/coinbasepro_websocket.go

```diff
@@ -515,3 +503,30 @@ func base64URLEncode(b []byte) string {
 	s = strings.ReplaceAll(s, "/", "_")
 	return s
 }
+
+// checkSubscriptions looks for incompatible subscriptions and if found replaces all with defaults
```

Comment:
1) Because the websocket isn't actually working right now, we can use any of the incompatible ones to spot that the  _entire_ config is completely out-of-date, and then upgrade-and-replace the whole thing, including adding new ones.

Equivalent to saying "You have config v1. That's now entirely unusable. Replacing it with v2".

Because Coinbase WS isn't working, it doesn't seem likely anyone has tried customising these subscriptions, so we're okay to just replace them wholesale.